packages/platform-android/native_modules.gradle

```diff
@@ -229,8 +230,8 @@ ext.applyNativeModulesSettingsGradle = { DefaultSettings defaultSettings, String
 ext.applyNativeModulesAppBuildGradle = { Project project, String root = ".." ->
   autoModules.applyBuildGradle(project, root)
 
-  def generatedSrcDir = "generated/rncli/src/main/java"
-  def generatedCodeDir = new File(buildDir, "$generatedSrcDir/com/facebook/react")
+  def generatedSrcDir = new File(buildDir, "generated/rncli/src/main/java/")
```

Comment:
do we need this trailing slash? other directories don't have it

Reply:
No, sry. I remove the slash.